src/main/java/seedu/address/model/person/NokPhone.java

```diff
@@ -0,0 +1,60 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's next of kin's phone number in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidNokPhone(String)}
+ */
+public class NokPhone {
+
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Phone numbers should only contain numbers, and it should be at least 3 digits long";
+    public static final String VALIDATION_REGEX = "\\d{3,}";
+    public final String value;
+
+    /**
+     * Constructs a {@code NokPhone}.
+     *
+     * @param nokPhone A valid next of kin phone number.
+     */
+    public NokPhone(String nokPhone) {
+        requireNonNull(nokPhone);
+        checkArgument(isValidNokPhone(nokPhone), MESSAGE_CONSTRAINTS);
+        value = nokPhone;
+    }
+
+    /**
+     * Constructs a {@code NokPhone} with default value "000".
+     */
+    public NokPhone() {
+        value = "000";
+    }
```

Comment:
Just curious is there any reason why the default value for NOK phone is "000"? I referenced to AB3's Phone.java class (which I think it should be similar) and I did not find any "000" there.

Reply:
Dummy value used here just a placeholder(to be updated subsequently in a separate PR for adding NokPhone). Also addresses comment by @sjoann.